docs/common-issues.rst

```diff
@@ -0,0 +1,31 @@
+Common Issues with ev3dev-lang-python
+=====================================
+
+``/usr/bin/env: 'python3\r': No such file or directory``
+--------------------------------------------------------
+
+This means your file includes Windows-style line endings
+(CRLF--carriage-return line-feed), which are often inserted by editors on
+Windows. To resolve this issue, open an SSH session and run the following
+command, replacing ``<file>`` with the name of the Python file you're
+using:
+
+.. code:: shell
+
+    sed -i 's/\r//g' <file>
+
+This will fix it for the copy of the file on the brick, but if you plan to edit
+it again from Windows, you should configure your editor to use Unix-style
+line endings (LF--line-feed). For PyCharm, you can find a guide on doing this
+`here <https://www.jetbrains.com/help/pycharm/2016.2/configuring-line-separators.html>`_.
+In Visual Studio Code, there is an option in the lower-right corner.
```

Comment:
I think most users are using VS Code these days, so maybe list this one first with a screenshot?